client/js/components/fragments/MiniProfile.js

```diff
@@ -1,19 +1,32 @@
 import React, { Component, PropTypes } from 'react';
 import { Link } from 'react-router';
 
-import DefaultAvatar from '../../../img/avatar.jpg';
-import 'file?name=[name].[ext]!../../../img/avatar.jpg';
-
 class MiniProfile extends Component {
   render() {
     const { user } = this.props;
+
+    let avatar = null;
+    if (user.avatar) {
+      avatar = (
+        <span className="media-left">
+          <img className="media-object img-circle" src={user.avatar} alt={`${user.firstName} ${user.lastName}`}/>
+        </span>
+      );
+    }
+
+    let name = (<p className="text-muted">{user.email}</p>);
+    if (user.slackId) {
+      name = (<p className="text-muted">Slack: @{user.slackId}</p>);
+    }
+
     return (
-      <span>
-        <Link to={`/profile/${user.id}`} title={'${user.firstName} ${user.lastName}'}>
-          <img src={user.avatar || DefaultAvatar} alt={'${user.firstName} ${user.lastName}'}/>
-          <p>{user.firstName} {user.lastName}</p>
-        </Link>
-      </span>
+      <Link className="miniProfile media" to={`/profile/${user.id}`} title={`${user.firstName} ${user.lastName}`}>
+          {avatar}
+        <span className="media-body">
+          <h4 className="media-heading">{user.firstName} {user.lastName}</h4>
+          {name}
```

Comment:
It's either email or slack ID so I wouldn't call it "name"  - I know, naming stuff is hard :). 
Could you also put <p></p> here and keep only content of the <p> in the variable? It's the same for both email and slack ID so no need to keep it in the variable.

So that in this line we have
<p className="text-muted">{whateverYouDecideToCallTheVariable}</p>